Add support for /api/v1/rules (#16)
openshift/prom-label-proxy: README.md

## How does this project work?

> What this proxy does is it proxies the `/federate`, `/api/v1/query`, `/api/v1/query_range`, `/api/v1/rules` Prometheus endpoints and ensures that a particular label is enforced in the particular request and response.

Reviewer: ```suggestion
This project is a proxy that proxies the `/federate`, `/api/v1/query`, `/api/v1/query_range`, `/api/v1/rules` Prometheus endpoints and ensures that a particular label is enforced in the certain request and response.
```

> What this proxy does is it proxies the `/federate`, `/api/v1/query`, `/api/v1/query_range`, `/api/v1/rules` Prometheus endpoints and ensures that a particular label is enforced in the particular request and response.

Reviewer: What about `/api/v1/alerts`?

Author: for another PR
